src/main/scala/org/globalforestwatch/summarystats/carbonflux_custom_area/CarbonCustomDF.scala

```diff
@@ -0,0 +1,116 @@
+package org.globalforestwatch.summarystats.carbonflux_custom_area
+
+import com.github.mrpowers.spark.daria.sql.DataFrameHelpers.validatePresenceOfColumns
+import org.apache.spark.sql.functions.{length, max, sum}
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+object CarbonCustomDF {
+
+  val contextualLayers: List[String] = List(
+    "treecover_density__threshold",
+    "carbon_flux_custom_area_1"
+  )
+
+  def unpackValues(df: DataFrame): DataFrame = {
+
+    implicit val spark: SparkSession = df.sparkSession
+    import spark.implicits._
```

Comment:
Codacy found an issue: [Put imports in the beginning of the package](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6075981)